pillar.example

```diff
@@ -1,16 +1,16 @@
 vault:
   version: 0.7.0
-  listen_protocol: tcp
-  listen_port: 8200
-  listen_address: 0.0.0.0
-  strict_tls: 0
-  tls_cert_file: {}
-  tls_key_file: {}
-  default_lease_ttl: 4380h
-  max_lease_ttl: 43800h
   self_signed_cert:
     enabled: false
-  backend: {}
   dev_mode: true
   service:
-    type: upstart
+    type: systemd
+
+  server:
+    listener:
+      - tcp:
+          address: "127.0.0.1:8200"
+          tls_disable: true
+    storage:
```

Comment:
You mean the `storage` dict? In [Vault configuration](https://www.vaultproject.io/docs/configuration/) it is specified that `storage` param is required.